proxy/api/src/http/project.rs

```diff
@@ -508,14 +516,34 @@ mod test {
         );
 
         let res = request()
-            .method("GET")
+            .method("POST")
             .path(&format!("/request/{}", urn))
             .reply(&api)
             .await;
 
-        // TODO(finto): Response should eventually be a Request.
+        let request = json!(request::SomeRequest::Created(request::Request::new(
+            urn,
+            Instant::now()
+        )));
+
+        // TODO(sos): It might be nice to test the exact fields like this, but
+        // not sure how we want to handle `Instant`s in the test context, esp. because
+        // they're not easily serializable on their own.
+
+        // let want = json!({
+        //     "Created": {
+        //         "attempts": {
+        //             "clones": 0,
+        //             "queries": 0
+        //         },
+        //         "state": {},
+        //         "timestamp": 1602031207235,
+        //         "urn": "rad:git:hwd1yreyfmtg5ntmm97g8eazmb1uh75xpbta8mdthkcj3ee3yxcci3d8b5y"
+        //     }
+        // });
+
         http::test::assert_response(&res, StatusCode::OK, |have| {
-            assert_eq!(have, json!(true));
+            assert_eq!(have, request);
```

Comment:
I wonder if we could deserialize the timestamp and use it above with creating the expected request, i.e. something along the lines of:

```rust
        let timestamp = Instant::deserialize(have["timestamp"]).unwrap();
        let request = json!(request::SomeRequest::Created(request::Request::new(
            urn,
            timestamp
        )));
```